src/cpp/session/resources/schema/user-prefs-schema.json

```diff
@@ -1213,6 +1213,12 @@
             "title": "Tab key always moves focus",
             "description": "Tab key moves focus out of text editing controls instead of inserting tabs."
         },
+        "find_panel_legacy_tab_sequence": {
+            "type": "boolean",
+            "default": false,
+            "title": "Tab key behavior in find panel matches RStudio 1.2 and earlier",
```

Comment:
I feel like we should describe the behavior in terms of what it does, as opposed to the version of RStudio it's associated with. What do you think about having the title match the description in this regard? "Tab key moves focus directly from find to replace" or similar?

Reply:
(mainly because this is the text that shows up in the command palette and so it's the text most users will see)

Reply:
True, it's super-bizarre the way I've got it. I'll try something more like what you're suggesting.